setup-yarn-cache/action.yml

```diff
@@ -38,6 +38,17 @@ runs:
         echo "result=false" >> $GITHUB_OUTPUT
       fi
 
+  - name: Symlink current action repo
+    env:
+      action_path: ${{ github.action_path }}
+    run: ln -fs ${{ env.action_path }}/.. .github/.linked-infra-global-github-actions
+    shell: bash
```

Comment:
This workaround is necessary as per https://github.com/orgs/community/discussions/41927